messages_appendix.cddl

```diff
@@ -75,23 +75,24 @@ auth-initiation-token = (
   ? 0 : text ; token
 )
 
-; type key 1002
-auth-spake2-need-psk = {
-  auth-initiation-token
-}
+auth-spake2-psk-status = &(
+ psk-needs-presentation: 0
+ psk-shown: 1
+ psk-input: 2
+)
 
-; type key 1003
+; type key 1002
 auth-spake2-handshake = {
-  auth-initation-token
-  1 : bytes .size 32 ; random
+  0: auth-spake2-psk-status ; psk-status
+  1: bytes ; public-value
 }
 
-; type key 1004
+; type key 1003
 auth-spake2-confirmation = {
-  1 : bytes .size 32 ; transcript-mac
+  1 : bytes .size 64 ; transcript-mac
```

Comment:
perhaps out of scope, but `transcript-mac` seems like an odd naming choice to me. MAC is the message authentication code algorithm right (as opposed to the more globally used media access control address). `transcript-mac` then corresponds to `Ke` or `Ka` right?

Maybe name this `mac-key` or `confirmation-key` then?